Assets/Scripts/Model/Content/SecondEdition/Pilots/Rebel/ASF01BWing/BlueSquadronPilot.cs

```diff
@@ -38,14 +38,17 @@ public class BlueSquadronPilotXWA : BlueSquadronPilot
     {
         public BlueSquadronPilotXWA() : base()
         {
-            (PilotInfo as PilotCardInfo25).Cost = 4;
-            (PilotInfo as PilotCardInfo25).LoadoutValue = 8;
+            (PilotInfo as PilotCardInfo25).Cost = 12;
```

Comment:
Should be cost 10 loadout 8 - think you got Blade and Blue squadron the wrong way round!